docs/docs/hardware.md

```diff
@@ -36,6 +36,7 @@ That being said, there are currently only a few specific [boards](/docs/faq#what
 - [QAZ](https://www.cbkbd.com/product/qaz-keyboard-kit) (`qaz`)
 - [CRBN](https://keygem.store/collections/group-buys/products/group-buy-featherlight-40-kit) (`crbn`)
 - [tidbit](https://nullbits.co/tidbit/) (`tidbit`)
+- [eek!](https://www.cbkbd.com/product/eek-keyboard) (`eek`)
```

Comment:
You added a `!` to the keyboard name setup descriptions.  Should this link also follow that style?

Also, I noticed you used a capital "Eek!" in setup and "eek!" in the keyboard name.